### PR TITLE
Package biocaml.0.10.1

### PR DESCRIPTION
**Pull request by pveber**
### `biocaml.0.10.1`
The OCaml Bioinformatics Library
Biocaml aims to be a high-performance user-friendly library for
Bioinformatics. See the [website](http://biocaml.org) for detailed API
documentation. We welcome contributors and feedback from users. Please
contact us with any comments and suggestions for features you would
like added.

---
* Homepage: http://biocaml.org
* Source repo: git+https://github.com/biocaml/biocaml.git
* Bug tracker: https://github.com/biocaml/biocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/biocaml/biocaml.0.10.1/opam [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "David Koppstein"
+  "Alexander Gryzlov"
+  "Francois Berenger"
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
+
+depends: [
+  "base64"
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/biocaml/biocaml.0.10.1/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

Comment (kit-ty-kate) [reply packages/biocaml/biocaml.0.10.1/opam]: Could you return this fix upstream?

Comment (pveber) [reply packages/biocaml/biocaml.0.10.1/opam]: Thanks for looking at it! 
This is done in https://github.com/biocaml/biocaml/commit/11c6e1a330ed9e923e002e87ac5a922821742837

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by pveber** [comment]
Thanks a lot!

**Pull request merged**